cryptoorgchain/chain.json

```diff
@@ -31,33 +31,33 @@
   },
   "codebase": {
     "git_repo": "https://github.com/crypto-org-chain/chain-main",
-    "recommended_version": "v4.2.2",
+    "recommended_version": "v4.2.6",
     "compatible_versions": [
-      "v4.2.2"
+      "v4.2.6"
```

Comment:
Assuming this patch is not consensus-breaking against v4.2.2, can we please keep v4.2.2 as a compatible version here (and also in the corresponding version object in the `versions` array)

Reply:
That's fine by me, but do we want to even suggest that full node versions with security vulnerabilities are compatible?

Reply:
That's a very good point, and there's no strong standard on that at this point. The simple answer is: yes, still compatible. An alternative off the top of my head is: to create a new version object that starts at the secure version, and point the previous and use the `next_version_name` property (which forms a linked list) to essentially skip the version with the vulnerability. Really, it's a topic that should be discussed further.